internal/host/monitor.go

```diff
@@ -35,16 +28,29 @@ func (m *Manager) initMonitoringQueryGenerator() {
 				models.HostStatusPreparingSuccessful,
 				models.HostStatusInstalling,
 				models.HostStatusInstallingInProgress,
-				models.HostStatusInstalled,
 				models.HostStatusInstallingPendingUserAction,
 				models.HostStatusResettingPendingUserAction,
-				models.HostStatusCancelled, // for limited time, until log collection finished or timed-out
-				models.HostStatusError,     // for limited time, until log collection finished or timed-out
 			}
 
-			dbWithCondition := common.LoadTableFromDB(db, common.HostsTable, "status in (?)", monitorStates)
-			dbWithCondition = common.LoadClusterTablesFromDB(dbWithCondition, common.HostsTable)
-			dbWithCondition = dbWithCondition.Where("exists (select 1 from hosts where clusters.id = hosts.cluster_id)")
+			// monitor following states for limited time, until log collection finished or timed-out
+			monitorStatesUntilLogCollection := []string{
+				models.HostStatusCancelled,
+				models.HostStatusError,
+			}
+			logCollectionEndStates := []string{
+				string(models.LogsStateCompleted),
+				string(models.LogsStateTimeout),
+				string(models.LogsStateEmpty),
+			}
+
+			dbWithCondition := common.LoadClusterTablesFromDB(db)
+			dbWithCondition = dbWithCondition.Where(
+				"id IN (SELECT cluster_id FROM hosts WHERE hosts.status in (?) OR (hosts.status in (?) AND hosts.logs_info not in (?)))",
+				monitorStates, monitorStatesUntilLogCollection, logCollectionEndStates)
+			dbWithCondition = dbWithCondition.Or(
+				"id IN (SELECT cluster_id FROM hosts WHERE clusters.id = hosts.cluster_id AND hosts.status = ? AND clusters.status <> ?)",
```

Comment:
This adds clusters with status != installed if they have an installed host.
note that prior to this change the clusters selected by this query would go through the `SkipMonitoring` filter but after this PR they don't